physionet-django/project/views.py

```diff
@@ -1909,6 +1918,8 @@ def sign_dua(request, project_slug, version):
     Page to sign the dua for a protected project.
     Both restricted and credentialed policies.
     """
+    from project.cloud.s3 import has_aws_credentials
```

Comment:
imports belong at the top of the module

Reply:
I've moved import to the top of the module.